.pylintrc

```diff
@@ -1,2 +1,2 @@
 [MASTER]
-disable=fixme,logging-fstring-interpolation
\ No newline at end of file
+disable=fixme,logging-fstring-interpolation,duplicate-code
```

Comment:
added "duplicate-code" here. might be dangerous, but without it pylint flags all the code (since the functionality is basically copy-pasted). but I don't think it it necessary to abstract functionality even further.

Reply:
Do you think we could put it in the files containing the warning directly? This does seem a too much of a blanket solution.